src/views/Home.vue

```diff
@@ -144,6 +154,19 @@ export default class Home extends Vue {
             ' <strong class="required">(' + requiredText + ")</strong>";
         }
 
+        if (options.question.help) {
+          let helpTxt = sender.locale == "fr" ? String(options.question.help.fr) : String(options.question.help.default);
+          helpTxt = helpTxt
+            .replace(/&/g, '&amp;')
+            .replace(/"/g, '&quot;')
+            .replace(/'/g, 'ooooo')
```

Comment:
Not sure this escape makes sense. I think this replace section should just be using something like https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent

Reply:
Good eye. This escape is similar to encodeURIComponent, but add some more escape, and work around.